src/sxastarter/package.json

```diff
@@ -42,7 +42,7 @@
     "font-awesome": "^4.7.0",
     "graphql": "~15.8.0",
     "graphql-tag": "^2.12.6",
-    "next": "^13.1.6",
+    "next": "^13.4.12",
```

Comment:
@sc-addypathania This version should still match our template.
https://github.com/Sitecore/jss/blob/release/21.0.0/packages/create-sitecore-jss/src/templates/nextjs/package.json#L35